zeppelin-zengine/src/test/java/org/apache/zeppelin/notebook/NotebookTest.java

```diff
@@ -1175,13 +1176,15 @@ public void testPublicPrivateNewNote() throws IOException, SchedulerException {
     
     // create private note
     Note notePrivate = notebook.createNote(new AuthenticationInfo("user1"));
+    notePrivate.setName("1111");
     
     // only user1 have notePrivate right after creation
     notes1 = notebook.getAllNotes(user1);
     notes2 = notebook.getAllNotes(user2);
     assertEquals(notes1.size(), 2);
     assertEquals(notes2.size(), 1);
-
+    assertEquals(notes1.get(0).getId(), notePrivate.getId());
```

Comment:
I think `setName` may not be the perfect solution because Id would be less than `1111`. How about modifying `note1.contains(notePrivate.getId())`?